### PR TITLE
[PKX-007] Pokemon Moves tab

### PR DESCRIPTION
**Pull request by giantiyan**

### CODE REVIEW

File: lib/features/moves_tab/moves_tab.dart [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    this.pokemonType,
+  });
+
+  final PokemonMovesModel? moves;
+  final String? pokemonType;
+
+  @override
+  Widget build(BuildContext context) {
+    return (moves != null)
+        ? Scrollbar(
```
Comment (ryle-roasa) [inline lib/features/moves_tab/moves_tab.dart]: I think the Scrollbar widget is unnecessary

File: lib/features/moves_tab/moves_tab.dart [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            child: SingleChildScrollView(
+              child: Padding(
+                padding: const EdgeInsets.all(10.0),
+                child: Wrap(
+                  children: moves?.name?.map((name) {
+                        return Type(
```
Comment (ryle-roasa) [inline lib/features/moves_tab/moves_tab.dart]: convert to expression body

File: lib/features/home_page/widgets/type.dart [hunk 3]
```diff
@@ -3,25 +3,31 @@ import 'package:pokedex/utilities/colors.dart';
...
             bottom: 8,
-            right: 8,
+            right: 4,
           ),
           decoration: BoxDecoration(
-            color: Colors.white.withOpacity(0.25),
+            color: pokemonType != null
+                ? pokemonType.toString().pokemonColor
+                : white.withOpacity(0.25),
             borderRadius: BorderRadius.all(Radius.circular(25)),
```
Comment (ryle-roasa) [inline lib/features/home_page/widgets/type.dart]: BorderRadius.circular(25)

File: lib/features/moves_tab/moves_tab.dart (hunk 1 above)
Comment (ryle-roasa) [reply lib/features/moves_tab/moves_tab.dart]: same in About tab

File: lib/features/about_tab/about_tab.dart [hunk 4]
```diff
@@ -12,40 +12,38 @@ class AboutTab extends StatelessWidget {
...
-                      ),
-                      AboutInformation(
-                        label: baseExperience,
-                        value: about?.base_experience.toString().toBaseExp,
-                      ),
-                    ],
+        ? SingleChildScrollView(
+          child: Padding(
+            padding: const EdgeInsets.all(18.0),
+            child: Container(
```
Comment (SaturnoMark) [inline lib/features/about_tab/about_tab.dart]: No need for container please remove

File: lib/features/base_stats_tab/base_stats_tab.dart [hunk 5]
```diff
@@ -19,27 +19,25 @@ class BaseStatsTab extends StatelessWidget {
...
-                          label: statNameList[index].baseStatLabel,
-                          value: base_stat.toString(),
-                          pokemonType: pokemonType,
-                        ),
-                      ),
-                    ],
+        ? SingleChildScrollView(
+          child: Padding(
+            padding: const EdgeInsets.all(18.0),
+            child: Container(
```
Comment (SaturnoMark) [inline lib/features/base_stats_tab/base_stats_tab.dart]: same here

**Review by SaturnoMark — CHANGES_REQUESTED** [review]

**Review by SaturnoMark — APPROVED** [review]

**Pull request merged**